### DISCUSSION TITLE
Traefik 2.5.5: Can't issue a certificate

### DISCUSSION THREAD

**Discussion by AudiWolf** [techno-tim/techno-tim.github.io#107] [Questions] [answered]
Getting the following errormessage and no certificate:
time="2021-12-19T21:55:26Z" level=error msg="Unable to obtain ACME certificate for domains \"xxx.xxx.xxx,*.xxx.xxx.xxx\" : unable to generate a certificate for the domains xxx.xxx.xxx *.xxx.xxx.xxx]: error: one or more domains had a problem:\n[*.xxx.xxx.xxx] [*.xxx.xxx.xxx] acme: error presenting token: cloudflare: failed to create TXT record: HTTP status 400: Record already exists. (81057)\n[xxx.xxx.xxx] [xxx.xxx.xxx] acme: error presenting token: cloudflare: failed to create TXT record: HTTP status 400: Record already exists. (81057)\n" providerName=cloudflare.acme

**Comment by timothystewart6** [comment, marked as answer]
Maybe try deleting the TXT record?  Also, what is the top level domain you are tying to create this for?

**Reply by AudiWolf** [reply]
Somehow it is working now. Thanks for your support.
